configman/config_manager.py

```diff
@@ -15,30 +15,48 @@
 # for convenience define some external symbols here - some client modules may
 # import these symbols from here rather than their origin definition location.
 # PyFlakes may erroneously flag some of these as unused
-from configman.command_line import command_line
-from configman.converters import to_string_converters
-from configman.config_exceptions import NotAnOptionError
-from configman.config_file_future_proxy import ConfigFileFutureProxy
-from configman.def_sources import setup_definitions
+from configman.commandline import (
```

Comment:
hmmm, i thought we had already merged the change that made the form of all the imports the same.   We discussed a month or two ago about when to move to the multiline import form and when not to.  I find that consistency is more readable than having two different forms of imports.